code/modules/antagonists/heretic/status_effects/debuffs.dm

```diff
@@ -214,3 +214,90 @@
 /datum/status_effect/heretic_lastresort/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_IGNORESLOWDOWN, TRAIT_STATUS_EFFECT(id))
 	owner.AdjustUnconscious(20 SECONDS, ignore_canstun = TRUE)
+
+
+
+/// Used by moon heretics to make people mad
+/datum/status_effect/moon_converted
+	id = "moon converted"
+	alert_type = /atom/movable/screen/alert/status_effect/moon_converted
+	duration = -1
+	status_type = STATUS_EFFECT_REPLACE
+	///used to track damage
+	var/damage_sustained = 0
+	///overlay used to indicate that someone is marked
+	var/mutable_appearance/moon_insanity_overlay
+	/// icon file for the overlay
+	var/effect_icon = 'icons/effects/eldritch.dmi'
+	/// icon state for the overlay
+	var/effect_icon_state = "moon_insanity_overlay"
+
+/atom/movable/screen/alert/status_effect/moon_converted
+	name = "Moon Converted"
+	desc = "THEY LIE, THEY ALL LIE!!! SLAY THEM!!! BURN THEM!!! MAKE THEM SEE THE TRUTH!!!"
+	icon_state = "lastresort"
+
+/datum/status_effect/moon_converted/on_creation()
+	moon_insanity_overlay = mutable_appearance(effect_icon, effect_icon_state, ABOVE_MOB_LAYER)
+	return ..()
```

Comment:
similar to Initialize, you want to setup the as parent before you setup as the child
```suggestion
/datum/status_effect/moon_converted/on_creation()
	. = ..()
	moon_insanity_overlay = mutable_appearance(effect_icon, effect_icon_state, ABOVE_MOB_LAYER)
```